src/renderers/index.ts

```diff
@@ -41,7 +43,7 @@ class HTMLWidget extends Widget {
    * If the node is visible, it is typeset.
    */
   onAfterAttach(msg: Message) {
-    typeset(this.node);
+    typeset(this.node.firstChild as HTMLElement);
```

Comment:
Since you appended a document fragment, there may be more than one child. It's probably better to still typeset `this.node` to get all of the children.